content/_releases/22/24.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Product Update - v22.24
+description: Details of the product update v22.24 from 21st of June 2022.
+layout: rel
+version: "22.24"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-06-21
+redirect_from:
+  - /releases/22.24.html
+---
+
+## Improvements and Updates
+
+### Change of Footer position
+
+To extend vertical space of the flow designer screen we moved the Footer to the
+bottom of side-menu. You can now see the platform version, Privacy Policy and ToS
+links by opening the menu.
+
+{% include img.html max-width="30%" url="/assets/img/RN/2224/RN2224-new-footer-design.png" title="New footer design" %}
+
+### Re-authentication for sensitive actions (final part)
+
+This is the 3rd and final part of our initiative to prevent accidental or malicious
+changes to sensitive data and follows on from releases [22.20](/releases/22/20#re-authentication-for-sensitive-actions) and [22.22](/releases/22/22#re-authentication-for-sensitive-actions-continuation).
+
+In this final part we extend re-authentication for sensitive actions to support
+cases when a user logs in using OIDC/SAML. The user will be redirected to OIDC/SAML
+provider for authentication.
+
+> **Again, please note**: You will be asked to re-authenticate once every 6 hours.
+
+This feature will work on all modern web browsers versions from:
+
+*   Chrome `66+`
+*   Firefox `63+`
+
+### Component development and custom Dockerfile
+
+To improve component deployment and build process we modernised the `appbuilder`
+and `apprunner` docker images to address potential vulnerabilities in the `Node.js`
+packages. All platform services, as well as the `Node.js` based integration components
+are built with these images.
+
+The modernisation brought some important changes which might affect you when you
+deploy the new version of your component code.
+
+*   You can use custom `Dockerfile` to define the build process in the `Node.js` components. It must be in the root directory of your component code.
+*   If you have used a custom `Dockerfile` you might have noticed that it was not used in the build process. Now the build process **will check for the existence of it and use it**. Which means if you have not updated your `Dockerfile` recently, chances are your component build might fail or the newly built component might not function as expected. **Check your `Dockerfile` before deploying or remove it from the component code if custom build parameters are not necessary.**
+*   We now support `Node.js` version 18.
+*   **IMPORTANT**: custom `Dockerfile` is not yet supported for Java components.
+
+## Fixed bugs
+
+*   Fixed the bug in the object and attachment handling service. Now the service will retry streams on failure and will handle errors properly.
+*   Fixed the user interface bug when the required field symbol (`*`) would not show after the metadata reload in case the mandatory fields were controlled by a checkbox.
```

Comment:
This needs reworking - very difficult to understand. Suggest sthg like below:

Mandatory fields in the flow designer views will be consistently displayed, even when the fields are check-boxes and even after reloading metadata.

Reply:
Decided to leave as it is.